dubbo-common/src/main/java/org/apache/dubbo/common/utils/TimeUtils.java

```diff
@@ -73,4 +73,29 @@ private static synchronized void startTicker() {
             isTickerAlive = true;
         }
     }
+
+    public static Long parseTimeoutToMills(String timeoutVal) {
```

Comment:
LRU2Cache has been updated with overwrite 'computeIfAbsent' method. This wasn't overridden before, and using it could lead to concurrency issues due to the lack of locking.